compiler/circle-opselector/driver/Driver.cpp

```diff
@@ -255,10 +365,17 @@ int entry(int argc, char **argv)
     std::cerr << "ERROR: No operator selected" << std::endl;
     exit(EXIT_FAILURE);
   }
-  // TODO implement node selections
+
+  // make module
+  auto new_module = std::make_unique<luci::Module>();
+  // make graph with selected nodes
+  auto new_graph = ::make_graph(selected_nodes);
+  // TODO set proper graph name
+
+  new_module->add(std::move(new_graph));
 
   // Export to output Circle file
-  assert(opselector::exportModule(module.get(), output_path));
+  assert(opselector::exportModule(new_module.get(), output_path));
```

Comment:
(beyond this PR) It would be better to show explicit error if there is a problem while exporting module. Otherwise, the user will try something more with wrong circle. It could be done in another PR.

Reply:
@glistening Actually, there is explicit condition in the function.
```cxx
bool exportModule(luci::Module *module, std::string &output_path)
{
  luci::CircleExporter exporter;

  luci::CircleFileExpContract contract(module, output_path);

  if (!exporter.invoke(&contract))
  {
    std::cerr << "ERROR: Failed to export '" << output_path << "'" << std::endl; // NOTE HERE
    return false;
  }

  return true;
}
```

Strictly speaking, `assert` is useless. I'll remove it.